src/py.py

```diff
@@ -0,0 +1,10 @@
+# shim for pylib going away
+# if pylib is installed this file will get skipped
+# (`py/__init__.py` has higher precedence)
```

Comment:
For the record, this is only partially true. In https://github.com/spack/spack/pull/38989, one of our users reported that pytest-html doesn't work for them even when they have both pytest and py installed. The issue turned out to be that while the above statement may be true when both pytest and py are installed in the same directory, if they are installed in different directories, the library that comes first in PYTHONPATH is used instead. No need to change anything in pytest IMO, just a warning in case other users report this issue. Hoping the rest of the Python ecosystem moves to deprecate and remove py.